forc-plugins/forc-client/src/op/deploy.rs

```diff
@@ -275,66 +275,98 @@ pub async fn deploy_pkg(
     let tx = Transaction::from(tx);
 
     let chain_id = client.chain_info().await?.consensus_parameters.chain_id();
+    let pkg_name = manifest.project_name();
```

Comment:
This can be moved inside of `create_deployment_artifact` to reduce duplication. Since `manifest` is already an argument, we can remove `pkg_name` as an argument.